src/dxtbx/model/experiment_list.py

```diff
@@ -1306,8 +1320,9 @@ def _create_imageset(records, format_class, format_kwargs=None):
     return imageset
 
 
-def _create_imagesequence(record, format_class, format_kwargs=None):
-    # type: (ImageMetadataRecord, Type[Format], Dict) -> dxtbx.imageset.ImageSequence
+def _create_imagesequence(
+    record: ImageMetadataRecord, format_class: Type[Format], format_kwargs: Dict = None
```

Comment:
I'm surprised this isn't flagged as invalid annotation?

Reply:
Good catch. From https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type:
> You can use the --no-implicit-optional command-line option to stop treating arguments with a None default value as having an implicit Optional[...] type. It’s possible that this will become the default behavior in the future.

So we should turn this on.